iceoryx_utils/include/iceoryx_utils/internal/posix_wrapper/mutex.hpp

```diff
@@ -56,7 +56,7 @@ class mutex
 {
   public:
     /// @brief the construction of the mutex can fail which will lead to a call
-    ///         to std::terminate which is alright for the moment since we are
+    ///         to programm termination which is alright for the moment since we are
```

Comment:
will lead to a call to program termination ... sounds somehow weird. Could you please rephrase it a bit
Also typo: program.